src/main/java/lotto/domain/LottoResult.java

```diff
@@ -1,39 +1,58 @@
 package lotto.domain;
 
-import java.util.HashMap;
-import java.util.Map;
+import lotto.domain.constant.LottoRank;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class LottoResult {
 
-    private static Map<LottoRank, Integer> lottoResult;
+    private static List<LottoNumberResult> lottoNumberResult;
```

Comment:
```suggestion
    private static List<LottoNumberResult> lottoNumberResults;
    
    static {
        lottoNumberResults.add(new LottoNumberResult(LottoRank.FOURTH, 0));
        lottoNumberResults.add(new LottoNumberResult(LottoRank.THIRD, 0));
        lottoNumberResults.add(new LottoNumberResult(LottoRank.SECOND, 0));
        lottoNumberResults.add(new LottoNumberResult(LottoRank.BONUS, 0));
        lottoNumberResults.add(new LottoNumberResult(LottoRank.FIRST, 0));
    }
```
어차피 lottoNumberResults가 static이라면 위와 같이 static 구문 활용해 구현하는 것도 가능함